R/zzz.R

```diff
@@ -100,6 +100,52 @@
   }
 }
 
+.onAttach <- function(libname, pkgname) {
```

Comment:
What do you think about doing this in `.onLoad()` instead?

Reply:
Thanks Daniel. Per slack conversation, let's go with `.onLoad` since there are many users that never explicitly attach reticulate that will benifit from the feature. 

This also avoids us needing to handle the situation where the package is attached after reticulate has already initialized Python.